gravityforms-zero-spam.php

```diff
@@ -97,7 +122,7 @@ public function add_key_field( $form ) {
 		$autocomplete = RGFormsModel::is_html5_enabled() ? ".attr( 'autocomplete', 'new-password' )\n\t\t" : '';
 
 		$script = <<<EOD
-jQuery( document ).on( 'submit.gravityforms', '.gform_wrapper form', function( event ) {
+jQuery( "#gform_{$form['id']}" ).on( 'submit', function( event ) {
```

Comment:
@malayladu Does the fix not work unless we use the jQuery selector? If so, please add a docblock mentioning that. If the GF hook does work, let’s keep it to prevent changing more than necessary.